### ISSUE TITLE
broken due to halloween update and major reworks to discord's core

### ISSUE THREAD

**Issue by zenfty** [AdaelynXIV/RoundedDark#20]
betterdiscord itself is fixed as of 1.9.5 but this theme is still broken

**Comment by SimplyAnotherAnon** [comment]
plz update

**Comment by AstudilloMarillo** [comment]
Yes please, this is my favourite theme and I really want to continue using it!

**Comment by akrossu** [comment]
> Yes please, this is my favourite theme and I really want to continue using it!

Here's a temporary solution for if this main repository is updated or not; https://github.com/AKR0SS/ModifiedRoundedDark

**Comment by zenfty** [comment]
Legend.

**Comment by AstudilloMarillo** [comment]
> Here's a temporary solution for if this main repository is updated or not; https://github.com/AKR0SS/ModifiedRoundedDark

It does seem to work, so thanks for sharing! I still seem to have this issue https://github.com/AdaelynXIV/RoundedDark/issues/19 using this method though. I really don't know what's causing that.
